### ISSUE TITLE
Organism Loops

### ISSUE THREAD

**Issue by bminixhofer** [MEH-Design/frix#6]
An array of contents can be specified for organisms.
One Organism is created for every element of the array. Possible example syntax: 
```
{{oOne as variable}}
{{start}}
<div>{{variable}}</div>
{{end}}
```

**Comment by MiniXC** [comment]
Proposed html-syntax:

page
````html
<organism type="container" contains="article"/>
````

container organism
````html
<p>Something important before each new organism.</p>
<hr>
<cms-organisms>
<p>Something important after each new organism.</p>
````

**Comment by MiniXC** [comment]
The only problem here is that it's hard to do a syntax (without adding something like ``<loop>``) that allows ``container`` to have elements that are only used once. Example (with ``loop``):

````html
<h1 cms-content="title"></h1>
<loop>
<p>Something important before each new organism.</p>
<hr>
<organism cms-content="organisms"><!-- other organisms -->
<p>Something important after each new organism.</p>
</loop>
````

**Comment by MiniXC** [comment]
````json
{
  article: {
    title ...
  },
  quote: {
    attr ...
  }
}
````

**Comment by MiniXC** [comment]
Final solution:

````html
<loop name="test">
  <target />
  <br />
</loop>
````

````js
{
  test: {
    article: {...},
    article: {...},
  }
}
````

**Comment by MiniXC** [comment]
Added test for this functionality in  9dda108.

**Comment by MiniXC** [comment]
Tests are not passing, we still have a bug somewhere (maybe even in the test itself).

**Comment by bminixhofer** [comment]
fixed with f63f11cbdca44dace1ebc72ff259c0f9aaede7ed
